demo/assets/tailwind.config.js

```diff
@@ -7,11 +7,23 @@ module.exports = {
     themes: [
       {
         light: {
-          ...require('daisyui/src/theming/themes').light,
+          ...require('daisyui/src/theming/themes').light
+          ,
           primary: '#1d4ed8',
           'primary-content': 'white',
           secondary: '#f39325',
-          'secondary-content': 'white'
+          'secondary-content': 'white',
+          info: '#dbeafe',
+          'info-content': '#2563eb',
+          'error-content': '#e11d48',
+          'base-200': '#f3f4f6',
+          'base-300': '#e5e7eb'
```

Comment:
I think that we should go with the daisyui defaults, but will change the look a bit in the light color, I don't think that this will be a problem, but it is your call, what do you prefer?

Reply:
I don't think this will be a problem either. Personally, I don't like the default colors in light mode, so let's make some adjustments.

I suggest adding it back to the following:

```js
light: {
  ...require('daisyui/src/theming/themes').light,
  primary: '#1d4ed8',
  'primary-content': 'white',
  secondary: '#f39325',
  'secondary-content': 'white'
}
```

Maybe we can add these adjustments to the setup guide as @pehbehbeh points out, so everyone gets the same result?